Apps/WebClient/src/ClientApp/src/store/modules/vaccinationStatus/getters.ts

```diff
@@ -17,6 +17,11 @@ export const getters: VaccinationStatusGetters = {
     error(state: VaccinationStatusState): BannerError | undefined {
         return state.public.error;
     },
+    publicVaccineRecordError(
+        state: VaccinationStatusState
+    ): BannerError | undefined {
+        return state.publicVaccineRecord.error;
+    },
```

Comment:
Nit: might be better to move this after `publicVaccineRecordIsLoading` just for organizational purposes.

Reply:
`publicVaccineRecordIsLoading` doesn't need to be moved, just `publicVaccineRecordError`.